test/plot-schema.json

```diff
@@ -10735,6 +10735,12 @@
      "editType": "ticks",
      "valType": "boolean"
     },
+    "shift": {
```

Comment:
This one appears to be on the `xaxis`. 
Yet we are not going to implement this feature for the `x` axes in this PR.
If that's the case, let's make the adjustments in the attribute file so that it would only be added to the `yaxis `.
Thanks!